packages/server/test/websockets.test.ts

```diff
@@ -670,3 +674,169 @@ test('regression - badly shaped request', async () => {
   rawClient.close();
   t.close();
 });
+
+describe('include "jsonrpc" in response if sent with message', () => {
+  test('queries & mutations', async () => {
```

Comment:
you could flatten this 

```suggestion
test('include "jsonrpc" in response if sent with message', async () => {
```

but it's not like the tests are very well architected so i don't mind :D